tests/ops/mapper/test_image_diffusion_mapper.py

```diff
@@ -0,0 +1,124 @@
+import os
+import shutil
+import unittest
+
+from data_juicer.core.data import NestedDataset
+from data_juicer.ops.mapper.image_diffusion_mapper import \
+    ImageDiffusionMapper
+from data_juicer.utils.mm_utils import SpecialTokens
+from data_juicer.utils.unittest_utils import DataJuicerTestCaseBase, SKIPPED_TESTS
+
+# Skip tests for this OP in the GitHub actions due to disk space limitation.
+# These tests have been tested locally.
+@SKIPPED_TESTS.register_module()
+class ImageDiffusionMapperTest(DataJuicerTestCaseBase):
+
+    data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..',
+                             'data')
+
+    cat_path = os.path.join(data_path, 'cat.jpg')
+    img3_path = os.path.join(data_path, 'img3.jpg')
+
+    hf_diffusion = 'CompVis/stable-diffusion-v1-4'
+    hf_blip2 = 'Salesforce/blip2-opt-2.7b'
+
+    # dir to save the images produced in the tests
+    output_dir = '../diffusion_output/'
```

Comment:
@HYLcool What do you think?

Reply:
merge done

Reply:
Looks like it's a directory to store the generated images for the unit test. This test is a local test that will be skipped in the GitHub Actions, so maybe that's fine. But I think it's better to use a directory path similar to the `data_path` above.

Reply:
@HYLcool  Do we need to unify the output paths for temporary files?

Reply:
This will be solved with #198.